README.md

```diff
@@ -56,10 +152,184 @@ These three files can be downloaded directly from `inputs` and `dataset` directo
 
 Refer to [installation section](#installation) to set-up DeezyMatch on your local machine. 
 
-:warning: In the following tutorials, we assume a directory structure specified in the [this section](#data-and-directory-structure-in-tutorials).
+:warning: In the following tutorials, we assume a directory structure specified in [this section](#data-and-directory-structure-in-tutorials).
  
  Written in the Python programming language, DeezyMatch can be used as a stand-alone command-line tool or can be integrated as a module with other Python codes. In what follows, we describe DeezyMatch's functionalities in different examples and by providing both command lines and python modules syntaxes.
 
+### Quick tour 
+
+In this "quick tour", we go through all the DeezyMatch main functionalities with minimal explanations. If you want to know more about each module, refer to the relevant part of README (also referenced in this section):
+
+* [Train a new model](#train-a-new-model):
+
+```python
+from DeezyMatch import train as dm_train
+
+# train a new model
+dm_train(input_file_path="./inputs/input_dfm.yaml", 
+         dataset_path="dataset/dataset-string-similarity_test.txt", 
+         model_name="test001")
+```
+
+* Plot the log file (stored at `./models/test001/log.txt` and contains loss/accuracy/recall/F1-scores as a function of epoch):
+
+```python
+from DeezyMatch import plot_log
+
+# plot log file
+plot_log(path2log="./models/test001/log.txt", 
+         dataset="t001")
```

Comment:
Should the argument `dataset` be called something like `output_name` or `plot_name`?

Reply:
Changed to `output_name`

Reply:
Thanks!